.github/workflows/release-50_publish-docker.yml

```diff
@@ -60,57 +72,35 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
 
-      - name: Prepare temp folder
-        run: |
-          TMP=$(mktemp -d)
-          echo "TMP=$TMP" >> "$GITHUB_ENV"
-          pwd
-          ls -al "$TMP"
-
-      - name: Fetch lib.sh from polkadot repo
-        working-directory: ${{ env.TMP }}
-        run: |
-          curl -O -L \
-            -H "Accept: application/vnd.github.v3.raw" \
-            https://raw.githubusercontent.com/paritytech/polkadot/master/scripts/ci/common/lib.sh
-
-          chmod a+x lib.sh
-          ls -al
-
-      - name: Fetch release artifacts based on final release tag
+      #- name: Fetch release artifacts based on final release tag TODO: this step will be needed when automated triggering will work
         #this step runs only if the workflow is triggered automatically when new release is published
-        if: ${{ env.EVENT_NAME == 'release' && env.EVENT_ACTION != '' && env.EVENT_ACTION == 'published' }}
-        run: |
-          mkdir -p release-artifacts && cd release-artifacts
-
-          for f in $BINARY $BINARY.asc $BINARY.sha256; do
-            URL="https://github.com/${{ github.event.repository.full_name }}/releases/download/${{ github.event.release.tag_name }}/$f"
-            echo " - Fetching $f from $URL"
-            wget "$URL" -O "$f"
-          done
-          chmod a+x $BINARY
-          cp -f ${TMP}/lib.sh .
-          ls -al
+        # if: ${{ env.EVENT_NAME == 'release' && env.EVENT_ACTION != '' && env.EVENT_ACTION == 'published' }}
+        # run: |
+        #   mkdir -p release-artifacts && cd release-artifacts
+
+        #   for f in $BINARY $BINARY.asc $BINARY.sha256; do
+        #     URL="https://github.com/${{ github.event.repository.full_name }}/releases/download/${{ github.event.release.tag_name }}/$f"
+        #     echo " - Fetching $f from $URL"
+        #     wget "$URL" -O "$f"
+        #   done
+        #   chmod a+x $BINARY
+        #   cp -f ${TMP}/lib.sh .
+        #   ls -al
 
       - name: Fetch rc artifacts or release artifacts based on release id
         #this step runs only if the workflow is triggered manually
         if: ${{ env.EVENT_NAME  == 'workflow_dispatch' }}
         run: |
-          . ${TMP}/lib.sh
+          . ./.gitlab/common/lib.sh
```

Comment:
That cannot be right :)
We made `TMP` for a good reason... I don't think `./.gitlab/common` can work if we keep things clean.

Reply:
What do you mean by  "if we keep things clean" ? :)

Reply:
`TMP` was meant as a temp folder, so we should not use `.gitlab/common` for that.

Reply:
yeah, but `./gitlab/common` is not used as temp folder here as well

Reply:
the `TMP` folder was used previously because the `lib.sh` file used to be downloaded from the `polkadot` repo as it did not exist in `cumulus`. Now when we have all at one place we do not needed.